packages/edit-post/src/components/secondary-sidebar/list-view-sidebar.js

```diff
@@ -111,62 +116,49 @@ export default function ListViewSidebar() {
 		handleToggleListViewShortcut
 	);
 
-	/**
-	 * Render tab content for a given tab name.
-	 *
-	 * @param {string} tabName The name of the tab to render.
-	 */
-	function renderTabContent( tabName ) {
-		if ( tabName === 'list-view' ) {
-			return (
-				<div className="edit-post-editor__list-view-panel-content">
-					<ListView dropZoneElement={ dropZoneElement } />
-				</div>
-			);
-		}
-		return <ListViewOutline />;
-	}
-
 	return (
 		// eslint-disable-next-line jsx-a11y/no-static-element-interactions
 		<div
 			className="edit-post-editor__document-overview-panel"
 			onKeyDown={ closeOnEscape }
 			ref={ sidebarRef }
 		>
-			<Button
-				className="edit-post-editor__document-overview-panel__close-button"
-				icon={ closeSmall }
-				label={ __( 'Close' ) }
-				onClick={ closeListView }
-			/>
-			<TabPanel
-				className="edit-post-editor__document-overview-panel__tab-panel"
-				ref={ tabPanelRef }
-				onSelect={ ( tabName ) => setTab( tabName ) }
-				selectOnMove={ false }
-				tabs={ [
-					{
-						name: 'list-view',
-						title: _x( 'List View', 'Post overview' ),
-						className: 'edit-post-sidebar__panel-tab',
-					},
-					{
-						name: 'outline',
-						title: _x( 'Outline', 'Post overview' ),
-						className: 'edit-post-sidebar__panel-tab',
-					},
-				] }
-			>
-				{ ( currentTab ) => (
-					<div
-						className="edit-post-editor__list-view-container"
-						ref={ listViewContainerRef }
-					>
-						{ renderTabContent( currentTab.name ) }
+			<Tabs onSelect={ ( tabName ) => setTab( tabName ) }>
+				<div className="edit-post-editor__document-overview-panel__header">
+					<Button
+						className="edit-post-editor__document-overview-panel__close-button"
+						icon={ closeSmall }
+						label={ __( 'Close' ) }
+						onClick={ closeListView }
+					/>
+					<Tabs.TabList ref={ tabsRef }>
+						<Tabs.Tab tabId="list-view">
+							{ _x( 'List View', 'Post overview' ) }
+						</Tabs.Tab>
+						<Tabs.Tab tabId="outline">
+							{ _x( 'Outline', 'Post overview' ) }
+						</Tabs.Tab>
+					</Tabs.TabList>
+				</div>
+
+				<Tabs.TabPanel
+					ref={ listViewContainerRef }
```

Comment:
Just noting that previously `listViewContainerRef` was passed to all tabpanels, while now it's only passed to the list view tabpanel.

`listViewContainerRef` is used for 3 purposes:
- `focusOnMountRef`: used to move focus when the component mountsl. Changes from this PR shouldn't change a regression, since the default tab is hardcoded to be `'list-view'`, and therefore the outline tabpanel doesn't need a ref
- `listViewRef`: used to handle focus inside the `'list-view'` tab. Again, this shouldn't be needed by the outline tab
- `setDropZoneElement`: used to set the HTML drop zone as a callback ref. This also doesn't seem related to the outline tab.

In short, it seems OK that we're passing `listViewContainerRef` only to the list view tab panel.

Reply:
Thank you for adding more context around this change for anyone looking at it later.